docs/manual/troubleshooting.tex

```diff
@@ -856,12 +856,17 @@ \subsection{Install prerequisites\label{building-prerequisites}}
 
 \begin{Verbatim}
 brew update
-brew install caskroom/cask/brew-cask
```

Comment:
Why isn't this required anymore?

Reply:
brew-cask is already installed if you have Homebrew. 
$ brew install caskroom/cask/brew-cask will resolve in 
Error: Cask 'brew-cask' is unavailable: {HOME}/Homebrew/Library/Taps/caskroom/homebrew-cask/Casks/brew-cask.rb' does not exist.

Reply:
Thanks for the information!